### DISCUSSION TITLE
Do not crash whole packer_compiled if a single plugin config function fails?(e.g. wrap them all with pcall)

### DISCUSSION THREAD

**Discussion by vatosarmat** [wbthomason/packer.nvim#544] [Ideas]
How about make `packer.nvim` more config-error tolerant this way? Just print error messages for failed plugins and load others anyway, so the user still can comfortably use `neovim` and fix their config.

**Comment by wbthomason** [comment]
This is already largely the case. I'm guessing you're running into this while loading a group of lazy-loaded plugins on the same trigger, where an error in the config for one can prevent the others from loading? Assuming so, I have a commit which changes this behavior, which was an oversight, to not break from the loader function on error. I'll push it as soon as Github fixes https://www.githubstatus.com/incidents/rmfrw9dfbtbp.

**Reply by wbthomason** [reply]
Update: this is 5bca0eb.

**Reply by vatosarmat** [reply]
The issue was that I was passing strings to `config` instead of functions. As I see in `compile.lua`, `packer` uses `try_loadstring`  only if `type(config_item) == 'function'` while `executable_string` is inserted in `packer_compiled` as is. Maybe `executable_string` taken directly from plugin spec should also be wrapped into some sort of crash protection?
